### PR TITLE
refactor(admin-system): refactor domains module

### PR DESCRIPTION
**Pull request by brannon-darby**
### Description
refactor(domains): refactor domains in the admin-system module

#### Related issues

- None

### Checklist

- [ ] New tests added or existing tests modified to cover all changes

### CODE REVIEW

File: modules/admin-system/src/domains/components/domain-form.component.ts [hunk 1]
```diff
@@ -2,33 +2,27 @@ import { Component, OnInit } from '@angular/core'
 import { Router } from '@angular/router'
 import { UiService } from '@colmena/admin-ui'
 
-import { DomainsService } from '../domains.service'
+import { Domain, DomainsService } from '../domains.service'
```
Comment (brannon-darby) [inline modules/admin-system/src/domains/components/domain-form.component.ts]: does domains.service export the Domain model?

Comment (brannon-darby) [inline modules/admin-system/src/domains/components/domain-form.component.ts]: yes, i see now that you added it

File: modules/admin-system/src/domains/components/domain-form.component.ts [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    private service: DomainsService,
+    private uiService: UiService,
+    private router: Router,
+    private route: ActivatedRoute,
+  ) { }
+
+  ngOnInit() {
+    this.formConfig = this.service.getFormConfig()
+    this.routeTitle = this.route.snapshot.data.title
+    if (this.routeTitle === 'Create') {
```
Comment (beeman) [inline modules/admin-system/src/domains/components/domain-form.component.ts]: We should make this depend on the incoming data, not on the route.

File: modules/admin-system/src/domains/domains.routes.ts [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+        redirectTo: 'list',
+        pathMatch: 'full',
+      },
+      {
+        path: 'list',
+        component: DomainListComponent,
+        data: { title: 'List' },
+      },
+      {
+        path: 'create',
```
Comment (beeman) [inline modules/admin-system/src/domains/domains.routes.ts]: Let's keep this wrapped in the DomainDetailComponent to get better visual consistency

**Pull request merged**